src/cpp/r/session/graphics/RGraphicsDevice.hpp

```diff
@@ -46,6 +46,8 @@ int getWidth();
 int getHeight();
 double devicePixelRatio();
 
+void playDisplayList();
```

Comment:
I think the name and exposure level is fine (all of these interfaces are essentially "internals to internals")